videos/management/commands/sync_missing_captions.py

```diff
@@ -212,7 +212,7 @@ def create_new_content(self, file_content, video):
         new_obj = WebsiteContent.objects.get_or_create(
             website=video.website,
             filename=filename,
-            dirpath=dirpath,
+            dirpath="content/resources",
```

Comment:
We should never hard code a value like this in `ocw-studio`. The `dirpath` value here should come from the `folder` property of the category defined for this content in the `WebsiteStarter` object. I know it seems unlikely this would change, but it is possible. The `folder` for resources is defined here: https://github.com/mitodl/ocw-hugo-projects/blob/1aaf287450546a621b8f2f839b17f0403cb89403/ocw-course-v2/ocw-studio.yaml#L113